network_params.yaml

```diff
@@ -80,6 +80,7 @@ network_params:
   samples_per_slot: 8
   custody_requirement: 1
   target_number_of_peers: 70
+  additional_preloaded_contracts: '{"0x123463a4B065722E99115D6c222f267d9cABb524": {"balance": "1ETH","code": "0x1234","storage": {},"nonce": 0,"secretKey": "0x"}}'
```

Comment:
Does this actually do anything? Why not leave it `{}`?

Reply:
Adds an example so people can see how to use it, but yeah default can be {} if needed

Reply:
Would add the example in the github tests

Reply:
added an example in github tests and removed from here